test/instrumentation/modules/redis.js

```diff
@@ -39,7 +39,7 @@ test(function (t) {
     t.end()
   })
 
-  var client = redis.createClient()
+  var client = redis.createClient('6379', process.env.REDIS_HOST)
```

Comment:
@watson fyi the suggested handling `var client = redis.createClient({host: process.env.REDIS_HOST})` failed for redis drivers <= 2.0.1.